src/test/java/com/EighthLight/app/TestBoard.java

```diff
@@ -160,8 +165,8 @@ void hasAPlayerOneReturnsTrueWhenAPlayerWinsByRow() {
         ArrayList expectedBoard = new ArrayList<>(Arrays.asList("X", "X", "X",
                                                                 "O", "O", 5,
                                                                  6, 7, 8));
-        Player playerOne = new Player("X");
-        testBoard.setSpaces(expectedBoard);
+        MockPlayer playerOne = new MockPlayer("X");
+        setBoardState(testBoard, expectedBoard);
         testBoard.setWinningCombos();
```

Comment:
It seems weird to need to call this explicitly. Why does this need to be part of the public contract?